src/dex/maverick-v2/maverick-v2.ts

```diff
@@ -318,7 +333,9 @@ export class MaverickV2 extends SimpleExchange implements IDex<MaverickV2Data> {
     tokenAddress: Address,
     limit: number,
   ): Promise<PoolLiquidity[]> {
-    const _tokenAddress = this.dexHelper.config.wrapETH(tokenAddress);
+    const _tokenAddress = this.dexHelper.config
+      .wrapETH(tokenAddress)
+      .toLowerCase();
 
     const pools = await this._queryPoolsAPI(SUBGRAPH_TIMEOUT);
```

Comment:
We need to cache it for some time, no need to query it for all tokens each time

Reply:
fixed